mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -587,6 +587,8 @@ impl MarketMakerIt {
         }
 
         // Just use the IP given in the `conf`.
+
+        use rand::Rng;
```

Comment:
If I'm not mistaken, `rand::Rng` is used within `target_arch = "wasm32"`, so we can just remove this line and `#[cfg(target_arch = "wasm32")]` from the same import at the beginning of the file:
https://github.com/KomodoPlatform/atomicDEX-API/blob/df964c5902e3e741f004734fd0e5f97435ff9a2f/mm2src/mm2_test_helpers/src/for_tests.rs#L8